packages/commonwealth/client/scripts/views/components/component_kit/CWRelatedCommunityCard/CWRelatedCommunityCard.scss

```diff
@@ -0,0 +1,105 @@
+@import '../../../../../styles/shared.scss';
+
+.RelatedCommunityCard {
+  padding: 24px;
+  display: flex;
+  align-items: center;
+  width: 506px;
+  border-radius: 6px;
+  border: 1px solid $neutral-100;
+  background: #FFF;
+
+  &:hover {
+    box-shadow: $elevation-2;
+  }
+
+  .content-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 16px;
+
+    .top-content {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      gap: 8px;
+
+      .header {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+
+        .Text {
+          color: $neutral-900;
+          letter-spacing: -0.18px;
+        }
+      }
+
+      .description {
+        color: $neutral-800;
+        font-family: $font-family-neue-haas-unica;
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 20px; 
+      }
+    }
+
+    .metadata {
+      display: flex;
+      gap: 8px;
+      align-items: flex-start;
+
+      .member-data {
+        display: flex;
+        align-items: center;
+        gap: 4px;
+
+        .count, .text {
+          color: #656167; // neutrals 500
```

Comment:
use var from the pallete instead of hex, here, and everywhere else in this pr

Reply:
@masvelio Oh! I left these as notes to myself about an inconsistency in the color palette. Jess and I will be resolving this in a later ticket. In the meantime, I've set them to the current value of `$neutral-500` using the appropriate variable.